app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HelloRailsReact</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
```

Comment:
- [ ] Kindly I recommend you to **update this line `<%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>` with `<%= javascript_include_tag "/assets/application.js", "data-turbo-track": "reload", defer: true %>`** because when I run the app I got the following error , updating with the suggestion will resolve the issues, So kindly update with the given suggestion. 
![image](https://github.com/DianaBeki/hello-rails-react/assets/93598189/2aa57202-d28a-477c-9978-83eec89f4a4e)